rewrite-java/src/main/java/org/openrewrite/java/tree/JavaType.java

```diff
@@ -302,6 +302,23 @@ public static Class build(int flagsBitMap,
             }
         }
 
+        /**
+         * Can be used to find a class from the flyweights by fully-qualified name. If more than one class exists for
+         * the name, the first one is returned. This method will not return matches for fully-qualified classes that
+         * have type parameters.
+         * @param fullyQualifiedName Fully-qualified name of the class to look up.
+         * @return A class from the flyweights, if it exists.
+         */
+        @Nullable
+        public static Class findClass(String fullyQualifiedName) {
+            synchronized (flyweights) {
```

Comment:
The places where you use this method could be simplified if you made the fullyQualifiedName parameter `@Nullable`: 
```suggestion
        public static Class findClass(@Nullable String fullyQualifiedName) {
            if(fullyQualifiedName == null) {
                return null;
            }
            synchronized (flyweights) {
```